src/tribler/core/components/metadata_store/restapi/tests/conftest.py

```diff
@@ -13,32 +14,45 @@
 
 @pytest.fixture
 def add_fake_torrents_channels(metadata_store):
-    infohashes = []
+    def sha1_hash(value: str) -> bytes:
+        byte_array = bytearray(value, encoding='utf-8')
+        return hashlib.sha1(byte_array).digest()
```

Comment:
**nitpick:** can be written a bit more concise:
```suggestion
        return hashlib.sha1(value.encode('utf8')).digest()
```

Reply:
Two-liner is easier to read than one-liner :)

Reply:
Sometimes two lines are indeed much clearer than one line. But I'm not sure this is that case. `sha1` accepts bytes, and the most straightforward way to convert string to bytes is to call the string's `encode` method. The difference between `bytearray` and `bytes` is that `bytearray` is mutable. So when someone creates `bytearray` instead of bytes, it can be expected that he wants to somehow change the value of the bytearray. Otherwise, it just looks strange.

Reply:
Ok, agree, thanks for the explanation.